intranet/apps/eighth/views/attendance.py

```diff
@@ -678,3 +680,41 @@ def sponsor_schedule_widget_view(request):
     context.update({"eighth_sponsor": eighth_sponsor})
 
     return render(request, "eighth/sponsor_widget.html", context)
+
+
+@login_required
+@deny_restricted
+def email_students_view(request, scheduled_activity_id):
+    scheduled_activity = get_object_or_404(EighthScheduledActivity, id=scheduled_activity_id)
+
+    if not scheduled_activity.user_is_sponsor(request.user) and not request.user.is_eighth_admin:
+        raise Http404
+
+    if request.method == "POST" and request.POST.get("body"):
+        emails = [signup.user.notification_email for signup in scheduled_activity.eighthsignup_set.all()]
+
+        subject = settings.EMAIL_SUBJECT_PREFIX + "{}: Message from {}".format(scheduled_activity, request.user.full_name)
+        if request.POST.get("subject"):
+            subject += ": " + request.POST["subject"]
+
+        body_text = """{} has requested to send you the following email regarding {}:
+
+{}""".format(
+            request.user.full_name, scheduled_activity.activity, request.POST["body"]
```

Comment:
I don't think we necessarily want to trust the user-provided content, even when we are not rendering on the website specifically. Maybe we should apply `safe_html`?

Reply:
This is an `EmailMessage`, not an `EmailMultiAlternatives`. The default mime type set is `text/plain`; mail clients should not be parsing it as HTML.